erpnext/www/product_list.py

```diff
@@ -30,6 +30,9 @@ def get_context(context):
 
 	context.title = item_group
 
+	context.item_group_map, context['delivery_date'] = get_item_group_map(item_group)
+
+def get_item_group_map(item_group, delivery_date=None):
```

Comment:
Very nice Farhan! Good reuse of code.
You can also probably reuse this code for single item row.